From 6093286cfc8f83f1b89096ebb1fbfa99330bc490 Mon Sep 17 00:00:00 2001
From: Thomas Cahuzac <thomasx.cahuzac@intel.com>
Date: Tue, 5 Feb 2013 18:49:44 +0100
Subject: [PORT FROM MAIN] [DO NOT UPSTREAM][AUDIO] Context awareness support

BZ: 85264

Add constants needed by the Context Awareness feature.

Change-Id: If2a122b2200e64cc3792b343a0e2dc7c483d9a97
Orig-Change-Id: I0a0d89750dd26b783070b59762c3b3a62948207c
Signed-off-by: Thomas Cahuzac <thomasx.cahuzac@intel.com>
Reviewed-on: http://android.intel.com:8080/90345
Reviewed-by: Wagner, David <david.wagner@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Gonzalve, Sebastien <sebastien.gonzalve@intel.com>
Tested-by: Dixon, CharlesX <charlesx.dixon@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 47de1bb..d51c571 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -113,6 +113,11 @@ __BEGIN_DECLS
 /* Remote BGM state */
 #define AUDIO_PARAMETER_KEY_REMOTE_BGM_STATE "bgm_state"
 
+/* Context awareness */
+#define AUDIO_PARAMETER_KEY_CONTEXT_AWARENESS_STATUS "context_awareness_status"
+#define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_ON "on"
+#define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_OFF "off"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

